zzz_generated/app-common/app-common/zap-generated/cluster-objects.cpp

```diff
@@ -0,0 +1,352 @@
+#include "cluster-objects.h"
```

Comment:
So this is not actually a generated file, right?  It will break workflows that remove everything in zzz_generated and regenerate from scratch....

Reply:
Have moved it to `app/common` to prevent breaking workflows, will move back once we get full code gen going.